dockerfiles/init/modules/codenvy/templates/rsyncbackup.sh.erb

```diff
@@ -25,9 +25,12 @@ if [[ $(stat -c %a /opt/rsync_key) != 600 ]]; then
     chmod 0600 /opt/rsync_key
 fi
 
+RSYNC_PATH=$(ssh -i /opt/rsync_key -l ${SRC_USER_NAME} -o StrictHostKeyChecking=no -p ${SRC_PORT} ${SRC_HOST} \
```

Comment:
maybe we have to rename $RSYNC_PATH to something else to make it more obvious to read, because RSYNC_PATH is not actually path to rsync you need this only to detect run with sudo or not. but in general looks good